docs/timelion/getting-started/timelion-conditional.asciidoc

```diff
@@ -0,0 +1,62 @@
+
+[source,text]
+----------------------------------
+.es(index=metricbeat-*, timefield='@timestamp', metric='max:system.memory.actual.used.bytes')
+----------------------------------
+
+image::images/timelion-conditional01.png[]
+{nbsp}
+
+Let’s create two thresholds to keep an eye on the amount of memory your machine is using. For the purposes of this tutorial, your warning threshold will be 12.5GB and your severe threshold will be 15GB. When the maximum amount of memory being utilized exceeds either of these thresholds, the series will be re-styled accordingly.
+
+NOTE: If the threshold values are too high or low for your machine, please adjust accordingly.
+
+To configure these two threshold values, you can utilize Timelions conditional logic. In this tutorial you will use `if()` to compare each point to a number, adjust the styling if the comparison is `true` and use the default styling if the comparison is `false`. Timelion offers six operator values for comparison.
+
+[horizontal]
+`eq`:: equal
+`ne`:: not equal
+`lt`:: less than
+`lte`:: less than or equal to
+`gt`:: greater than
+`gte`:: greater than or equal to
+ 
+Since there are two thresholds, it makes sense to style them differently. Use the `gt` operator to color the warning threshold yellow with `.color('#FFCC11')` and the severe threshold red with `.color('red')`. Enter the following expression into the Timelion query bar to apply the conditional logic and threshold styling:
+
+[source,text]
+----------------------------------
+.es(index=metricbeat-*, timefield='@timestamp', metric='max:system.memory.actual.used.bytes'), .es(index=metricbeat-*, timefield='@timestamp', metric='max:system.memory.actual.used.bytes').if(gt,12500000000,.es(index=metricbeat-*, timefield='@timestamp', metric='max:system.memory.actual.used.bytes'),null).label('warning').color('#FFCC11'), .es(index=metricbeat-*, timefield='@timestamp', metric='max:system.memory.actual.used.bytes').if(gt,15000000000,.es(index=metricbeat-*, timefield='@timestamp', metric='max:system.memory.actual.used.bytes'),null).label('severe').color('red')
+----------------------------------
+
+image::images/timelion-conditional02.png[]
+{nbsp}
+
+For additional information on Timelions conditional capabilities, check out the blog post https://www.elastic.co/blog/timeseries-if-then-else-with-timelion[I have but one .condition()].
+
+Now that you have thresholds defined to easily identify outliers, let’s create a new series to determine what the trend really is. Timelions `mvavg()` function allows you to calculate the moving average over a given window. This is especially helpful for noisey time series. For this tutorial, you will use `.mvavg(10)` to create a moving average with a window of 10 data points. Use the following expression to create a moving average of the maximum memory consumption:
```

Comment:
we use 'memory consumption' here, but for consistent language, should we say 'memory usage'?